Octokit/Models/Request/RepositoryHookTestRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Octokit
+{
+    public class RepositoryHookTestRequest
```

Comment:
If we don't need to send any data with the payload for a POST, can we live without this class?